.github/workflows/i18nCheck.yml

```
@@ -18,7 +18,7 @@ jobs:
    
      - name: Setup tools
        run: |
-         install.packages(c("cli", "potools"))
+         install.packages(c("cli", "potools"), NSTALL_opts = c('--no-lock'))
```

```suggestion
         install.packages(c("cli", "potools"), INSTALL_opts = c('--no-lock'))
```
is this necessary though?